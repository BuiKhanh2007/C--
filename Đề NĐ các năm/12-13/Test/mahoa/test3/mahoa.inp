3
mrmtxgsxw jnyxx
mxkderuuiedfpnk
bygidynjzqie wj
 stlqxwyl jrwu 
yczlprjryq rb  
u lflrxmcft bse
sjlxufdqrjuq d 
 rndmjxl fln pi
dwyedgbk iz   z
eix n cifwzz jl
 yqyjgkxediyupl
l bbt sdjsc qyy
e cj xmj uwmljg
df ez sddxrkbxg
dquqejj qfdqk l
