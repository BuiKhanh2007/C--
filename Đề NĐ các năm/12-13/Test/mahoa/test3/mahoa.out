pupw{jv{z mq|{{
p{nghuxxlhgisqn
e|jlg|qm}tlh zm
 vwot{z|o muzx 
|f}osumu|t ue  
x oiou{pfiw evh
vmo{xigtumxt g 
 uqgpm{o ioq sl
gz|hgjen l}   }
hl{ q fliz}} mo
 |t|mjn{hgl|xso
o eew vgmvf t||
h fm {pm xzpomj
gi h} vgg{une{j
gtxthmm tigtn o
